config.defaults.yaml

```diff
@@ -6,41 +6,152 @@ kowalski:
   collections:
     gaia: Gaia_EDR3
     sources: ZTF_sources_20201201
+
 taxonomy:
-  name: Class type
-  children:
-    - name: phenomenological
-      children:
-        - name: variable
-        - name: periodic
-        - name: long timescale
-        - name: irregular
-        - name: eclipsing
-          children:
-            - name: EA
-            - name: EB
-            - name: EW
-        - name: flaring
-        - name: bogus
-    - name: ontological
-      children:
-        - name: pulsator
-          children:
-            - name: Delta Scuti
-            - name: Cepheid
-            - name: RR Lyrae
-            - name: LPV
-              children:
-                - name: Mira
-                - name: SRV
-        - name: binary star
-          children:
-            - name: W Uma
-            - name: Beta Lyr
-            - name: RS CVn
-        - name: AGN
-        - name: YSO
-        - name: CV
+  class: tds
+  name: time domain source type
+  subclasses:
+    - class: phenomenological
+      name: phenomenological
+      description: |
+        What can be said about an object given just its ZTF light curve.
+      subclasses:
+        - class: vnv
+          name: variable
+          description: |
+            Any ZTF source that shows detectable variability in its light curve
+            due to an astrophysical origin. This excludes variability due to blended photometry,
+            bright nearby stars, or any CCD artifact.
+        - class: pnp
+          name: periodic
+          description: |
```

Comment:
Should there be a cut on significance? Or any other criteria? I know that we follow a complex procedure. I was wondering what would be a good place to capture that.

Reply:
Not sure. @mcoughlin what say you, my friend?

Reply:
We have some rules of thumb, but the great thing about pnp is it seems to have solved the significance as a function of period problem. So not something simple we would include (we use ~10 for scanning the folders with Kevin, but that focuses on a single period range).